HISTORY.md

```diff
@@ -7,6 +7,7 @@
 ### Bug Fixes
 * fs_posix.cc GetFreeSpace() always report disk space available to root even when running as non-root.  Linux defaults often have disk mounts with 5 to 10 percent of total space reserved only for root.  Out of space could result for non-root users.
 * Subcompactions are now disabled when user-defined timestamps are used, since the subcompaction boundary picking logic is currently not timestamp-aware, which could lead to incorrect results when different subcompactions process keys that only differ by timestamp.
+* Fix an issue that `DeleteFilesInRange()` may cause ongoing compaction reports corruption exception, or ASSERT for debug build. There's no actual data loss or corruption that we find.
```

Comment:
Agree with listing it in 6.22.0 as I will backport this to 6.22.fb branch before releasing it anywhere.

Reply:
oh, I should create a new section. I'll leave it there as it will be backport to 6.22.0. thanks.